### PR TITLE
Create users

### PR DESCRIPTION
**Pull request by meeshysu**
funnyandy

Alright I think this is good to go!

### CODE REVIEW

File: src/components/pages/FriendItem/FriendItem.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import friendShape from '../../../helpers/propz/friendShape';
+import './FriendItem.scss';
+
+class FriendItem extends React.Component {
+  static propTypes = {
+    listing: friendShape,
```
Comment (zoeames) [inline src/components/pages/FriendItem/FriendItem.js]: listing??? awkward copy paste

Comment (meeshysu) [reply src/components/pages/FriendItem/FriendItem.js]: LOL, promise it wasn't copied with my mouse, just my fingers. I'll change that. :)

Comment (meeshysu) [reply src/components/pages/FriendItem/FriendItem.js]: Mkay I fixed my mistake in the latest commit!

**Comment by zoeames** [comment]
read friends and update user look good

**Pull request merged**